api/types.proto

```diff
@@ -27,6 +27,9 @@ message Resources {
 
 	// Amount of memory in bytes.
 	int64 memory_bytes = 2;
+
+	// Container pid limit
+	int64 pids_limit = 3;
```

Comment:
If this is a common pool that we orchestrate, then it should go here. If the orchestrator doesn't care and this enforced by the container, it should go there.

Also, `Resources` should be able to act as both limits and reservations, so the `_limit` suffix should be removed.